packages/commonwealth/client/index.html

```diff
@@ -3,27 +3,22 @@
   <head>
     <meta charset="utf-8" />
     <title>Common</title>
-    <base href="/" />
     <link rel="stylesheet" href="https://use.typekit.net/axy6pzh.css" />
-    <link rel="icon" type="image/x-icon" href="/favicon.ico?v=3" />
+    <link rel="icon" type="image/x-icon" href="/brand_assets/favicon.ico" />
```

Comment:
these are still not directly accessible via URL ex: `http://localhost:8080/brand_assets/32x32.png`.

unsure if this would break in production?

Reply:
This won't break, it seems to pull it fine locally:
<img width="1052" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/10c2ca6a-57e1-4020-8ee1-5d310ec36bb5">

<img width="1144" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/621636d8-8349-473c-8ae5-740714583640">

Reply:
ah, all good then. Thanks